hermes/webapp/src/index.html

```diff
@@ -12,10 +12,11 @@
         <div id="header">Hermes</div>
         <div style="padding: 10px" ng-view></div>
     </div>
-
     <script src="/vendor/angular/angular.js"></script>
     <script src="/vendor/angular-route/angular-route.js"></script>
     <script src="/vendor/angular-animate/angular-animate.js"></script>
+    <!--<script src="/vendor/sigma.js/sigma.min.js"></script>-->
```

Comment:
Ah yes.  I implemented all of this in Sigma but I couldn't get it work like I wanted to so I ripped Sigma out and replaced it but missed this line.